e2eAIOK/DeNas/nlp/bert_trainer.py

```diff
@@ -35,6 +39,15 @@ def _pre_process(self):
             custom_ops_thop = customer_ops_map_thop()
             macs_thop, _ = profile(self.model, inputs=(inputs,), custom_ops=custom_ops_thop)
             logging.info("(THOP) MACs: %.2f" % (macs_thop/(1000**3)))
+        if self.cfg.is_tl:
```

Comment:
this argument name is hard to relate to is_transferlearning, @tianyil1 , can you come up with a more easy to understand name so user will recognize what this argument is used for.

Reply:
Got it. I will change the name to "is_transferlearning".

Reply:
@tianyil1 , is_transferlearning is just I trying to understand 'is_tl', after reading your codes, I am thinking since you also used self.cfg.teacher_model, how about simply using if 'self.cfg.teacher_model' is none or sth to indicate 'is_tl'?